### PR TITLE
robocopy: add German translation

### PR DESCRIPTION
**Pull request by ZickZakk**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** -

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Can you translate paths too?

**Comment by ZickZakk** [comment]
> Can you translate paths too?
> 
> > Welcome to tldr.

Hello, thank you! 😄 
Good catch on the paths, I translated them as well.

File: pages.de/windows/robocopy.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# robocopy
+
+> Robustes Kopieren von Dateien und Ordnern.
+> Standardmäßig werden Dateien nur kopiert, wenn die Quell- und Zieldatei ein unterschiedliches Änderungsdatum oder eine unterschiedliche Dateigröße haben.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/robocopy>.
```
Comment (kbdharun) [inline pages.de/windows/robocopy.md:5]:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/robocopy>.
```
We don't specify l10n in Microsoft URLs from what I know the page automatically redirects you to your native page based on your language setting.

Comment (ZickZakk) [reply pages.de/windows/robocopy.md:5]: Just checked and you are right on that. Fixed the URL With your suggestion.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**